lib/stream.rb

```diff
@@ -9,4 +9,5 @@ module Stream
   require 'stream/person'
   require 'stream/public'
   require 'stream/tag'
+  require 'stream/bookmarked'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.